htsget-http-actix/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+
+[package]
+name = "htsget-http-actix"
+version = "0.1.0"
+authors = ["Daniel del Castillo de la Rosa <delcastillodelarosadaniel@gmail.com>"]
```

Comment:
I guess this should be the UMCCR group

Reply:
Are there any examples for that? I put my name because the other crates did that. 

Would "UMCCR <info@umccr.org>" be okay?

Reply:
I think it's fine to leave it at that, we are not a big corp and we acknowledge the original authors ;)

Reply:
Perfect!